File: mvc/src/main/java/com/bootcamp/mvc/model/Cuenta.java
package com.bootcamp.mvc.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Cuenta {
    private Long id;

    private double amountToken;

    private String tokenName;

    private Long id_wallet;

}

File: SegurityAngular-master/src/app/model/Cuenta.ts
export class Cuenta {
    id?:number;
    amount_tokens:number;
    tokenName:String;
    id_wallet:number;

    constructor(amountToken: number,tokenName:String,id_wallet:number){
    this.amount_tokens=amountToken;
    this.id_wallet=id_wallet;
    this.tokenName=tokenName;
    }
}